projects/packages/search/src/dashboard/components/record-meter/lib/record-info.js

```diff
@@ -55,12 +55,13 @@ export default function getRecordInfo( post_count, post_type_breakdown, tier, la
 	if ( numItems > 0 && hasValidData && hasBeenIndexed ) {
 		for ( let i = 0; i < numItems; i++ ) {
 			const theData = Object.values( post_type_breakdown )[ i ];
```

Comment:
Can we pick a more descriptive name for `theData`? Maybe `postTypeDetails` or similar?

Reply:
Totally! `postTypeDetails` works for me 👍